packages/expo-camera/src/Camera.types.ts

```diff
@@ -387,6 +371,30 @@ export type CameraProps = ViewProps & {
    * @platform ios
    */
   responsiveOrientationWhenOrientationLocked?: boolean;
+  /**
+   * A string representing aspect ratio of the preview, eg. `4:3`, `16:9`.
+   * @default 4:3
+   * @platform android
+   */
+  ratio?: CameraRatio;
+  /**
+   * Callback invoked when camera preview has been set.
+   */
+  onCameraReady?: () => void;
+  /**
+   * Callback invoked when camera preview could not been started.
+   * @param event Error object that contains a `message`.
+   */
+  onMountError?: (event: CameraMountError) => void;
+  /**
+   * Callback that is invoked when a barcode has been successfully scanned. The callback is provided with
+   * an object of the [`BarcodeScanningResult`](#barcodescanningresult) shape, where the `type`
+   * refers to the barcode type that was scanned and the `data` is the information encoded in the barcode
+   * (in this case of QR codes, this is often a URL). See [`BarcodeType`](#barcodetype) for supported values.
+   * for supported values.
```

Comment:
```suggestion
   * Callback that is invoked when a barcode has been successfully scanned. The callback is provided with
   * an object of the [`BarcodeScanningResult`](#barcodescanningresult) shape, where the `type`
   * refers to the barcode type that was scanned, and the `data` is the information encoded in the barcode
   * (in this case of QR codes, this is often a URL). See [`BarcodeType`](#barcodetype) for supported values.
   * for supported values.
```

Reply:
@amandeepmittal no, it's not necessary here. In the old package it needed to be explicitly set or you would get weird behaviour. The default now is the `CameraX` default `4:3`.